src/collective/excelexport/view.py

```diff
@@ -140,17 +146,15 @@ def _format_render(self, render):
         elif not isinstance(render, unicode):
             render = unicode(render)
 
-        return render.encode(encoding=self.encoding)
+        try:
+            return render.encode(encoding=self.encoding)
+        except UnicodeEncodeError:
+            # try default encoding
```

Comment:
How can a str(render) succeed (ascii encoding) if the previous (probably utf-8 or iso) failed?